vectordb_bench/backend/clients/elastic_cloud/elastic_cloud.py

```diff
@@ -145,7 +147,16 @@ def search_embedding(
 
     def ready_to_search(self):
         """ready_to_search will be called between insertion and search in performance cases."""
-        pass
+        assert self.client is not None, "should self.init() first"
+        self.client.indices.refresh(index=self.indice)
+        force_merge_task_id = self.client.indices.forcemerge(index=self.indice, max_num_segments=1, wait_for_completion=False)['task']
```

Comment:
Looks like we turn the `wait_for_completion` into False. Just curios why we don't make it a sync call by turning into True?

Reply:
> Just curios why we don't make it a sync call by turning into True?

This can take many minutes. Its not worth keeping the connection open.